src/browser/page.ts

```diff
@@ -192,6 +192,22 @@ export class Page {
     if (opts?.delay) {
       await new Promise((resolve) => setTimeout(resolve, opts.delay));
     }
+
+    if (!this.disableMemory) {
+      const state: ObjectiveState = {
+        kind: "ObjectiveState",
+        objective: "navigate to " + url,
+        url: url,
+        ariaTree: "about:blank",
+        progress: [],
+      };
+      const action: ModelResponseType = {
+        command: [{ kind: "GoTo", url }],
+        description: "Navigated to " + url,
+        progressAssessment: "",
+      };
+      await memorize(state, action, this.pageId);
```

Comment:
won't this just, like, pollute the index? (and still leak metadata given we can see *some* query is going on?)